datasets/librispeech/_metadata.json

```diff
@@ -0,0 +1,254 @@
+{
+    "@context": {
+        "@vocab": "https://schema.org/",
+        "ml": "http://mlcommons.org/schema/"
+    },
+    "@type": "Dataset",
+    "@language": "en",
+    "name": "LibriSpeech ASR corpus",
+    "url": "http://www.openslr.org/12",
```

Comment:
Maybe add `license`, to encourage adding this field? LibriSpeech is
`"license": "https://creativecommons.org/licenses/by/4.0/",` (see https://www.openslr.org/12)